### ISSUE TITLE
How to install using HACS

### ISSUE THREAD

**Issue by bartandeweg** [kuchel77/diskspace#2]
Sorry. But I probably missing something...

I've installed the plugin thru HACS. Now the this note is on top: 

> This integration supports config_flow, that means that you now can go to the integration section of your UI to configure it.
> No restart required 

Great! but what to do when i'm at the integration section. I cant find the integration in the list? What am i missing here

**Comment by kuchel77** [comment]
No, that was an error left in one of the config files that I didn't realise. 

I have fixed this hopefully in v0.5 so that it doesn't offer the option. You need to add this manually still to your configuration.yaml for it to work.
